packages/components/src/components/Spacer/Spacer.tsx

```diff
@@ -1,25 +1,45 @@
 import { View } from 'react-native'
 import React, { FC } from 'react'
 
+import { spacing } from '@department-of-veterans-affairs/mobile-tokens'
+
 export type SpacerProps = {
-  /** Size of the spacer, default 10 */
-  size?: number
+  /** Size of the spacer, default sm */
+  size?:
+    | 'none'
+    | '2xs'
+    | 'xs'
+    | 'sm'
+    | 'md'
+    | 'lg'
+    | 'xl'
+    | '2xl'
+    | '3xl'
+    | '4xl'
+    | '5xl'
+    | '6xl'
   /** True for horizontal spacing */
   horizontal?: boolean
 }
 
 /**
  * Convenience component for handling spacing without managing margin/padding between elements
- * @param size - Size of the spacer, default 10
+ * @param size - Size of the spacer, default sm (12)
  * @param horizontal - True for horizontal spacing
  * @returns A non-visible component sized to provide appropriate spacing
  */
-export const Spacer: FC<SpacerProps> = ({ size = 10, horizontal = false }) => {
+export const Spacer: FC<SpacerProps> = ({
+  size = 'sm',
+  horizontal = false,
+}) => {
+  const index = `vadsSpace${size[0].toUpperCase()}${size.slice(1)}`
+  const spacerSize: number = spacing[index as keyof typeof spacing]
```

Comment:
Side thought on this:

I thought this was kind of messy, but poking at it a bit I couldn't find a better way.

Was disappointed that:
```
  const index: keyof typeof spacing = `vadsSpace${size}`
  const spacerSize: number = spacing[index]
```
doesn't work because of the case on `size` (while agreeing it should remain lower case for consistency with our arguments/props generally).

Probably more effort than it's worth, but feels like there is _a way_ to simplify/improve this. But don't feel strongly enough to insist on it.

Reply:
The other option would to be to create a map which would be more type safe but less concise. Something like this:

```ts
const spacerSizeToSpacingKey = {
  'none': 'vadsSpaceNone',
  '2xs': 'vadsSpace2xs',
  'xs': 'vadsSpaceXs',
  'sm': 'vadsSpaceSm',
  'md': 'vadsSpaceMd',
  'lg': 'vadsSpaceLg',
  'xl': 'vadsSpaceXl',
  '2xl': 'vadsSpace2xl',
  '3xl': 'vadsSpace3xl',
  '4xl': 'vadsSpace4xl',
  '5xl': 'vadsSpace5xl',
  '6xl': 'vadsSpace6xl',
};
```